### PR TITLE
Fix Docker flag error

### PR DESCRIPTION
**Pull request by junxu**
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7071)

### CODE REVIEW

File: docs/setup/independent/install-kubeadm.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Docker. To ensure compatability you can either update Docker, like so:
 ```bash
 cat << EOF > /etc/docker/daemon.json
 {
-  "exec-opts": ["native.cgroupdriver=systemd"]
+  "exec-opt": ["native.cgroupdriver=systemd"]
```
Comment (tengqm) [inline docs/setup/independent/install-kubeadm.md]: According to docker documentation here: https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file, the examples are showing that "exec-opts" is the correct key. Have you tried it yourself and verified if this change works?

Comment (zhangxiaoyu-zidif) [reply docs/setup/independent/install-kubeadm.md]: `exec-opts` is correct.

Comment (junxu) [reply docs/setup/independent/install-kubeadm.md]: In my test env with docker 1.12.6, exec-opt is correct. https://docs.docker.com/engine/reference/commandline/dockerd/ according to dockerd cmd option, it's also "exec-opt"

Comment (tengqm) [reply docs/setup/independent/install-kubeadm.md]: unfortunately, there seem some inconsistency in docker documentation. when used in command line, the options is named `exec-opt`. however, when used in the json config file, the example shows `exec-opts`. it IS confusing.

Comment (zhangxiaoyu-zidif) [reply docs/setup/independent/install-kubeadm.md]: Here is json configuration, we should comply with json's format.

Comment (junxu) [reply docs/setup/independent/install-kubeadm.md]: But we con't start docker service with this json configuration, should we comply with a wrong configuration?

Comment (tengqm) [reply docs/setup/independent/install-kubeadm.md]: @junxu Okay, if you have tested that `exec-opt` works and `exec-opts` doesn't. We can merge this.

Comment (junxu) [reply docs/setup/independent/install-kubeadm.md]: Sorry, exec-opts works well.

**Pull request closed without merging**